doc/index.md

```diff
@@ -1,16 +1,74 @@
----
-sd_hide_title: true
----
+# Panel: A high-level app and dashboarding solution for Python
 
-```{image} _static/logo_stacked.png
----
-width: 125
----
+## Description
+
+Panel is a an [open-source](https://github.com/holoviz/panel/blob/main/LICENSE.txt) Python library that **makes it easy to create interactive dashboards and data apps using the tools you already know and love**.
+
+With Panel, you can quickly prototype and iterate on apps and dashboards using your favorite editor or notebook environment. Unlike other tools, Panel supports a wide range of visualization libraries and development environments, which means **you can choose the tools that work best for your workflow**.
+
+One of the most powerful features of Panel is its ability to handle large amounts of data and stream it directly to the frontend of your app. This allows you to build highly interactive and responsive apps that can handle real-time data updates with ease. Panel also provides deep bi-directional interactivity between the frontend and backend, which means **you can build complex and interactive apps that respond to user input in real-time**.
+
+In addition to these core features, Panel provides a number of **advanced features** that make it a unique and powerful tool for building data apps and dashboards. For example, **you can develop server-, pyodide-, and pyscript-backed applications, wrap your applications in polished templates for final deployment, and add authentication to your applications using the built-in OAuth providers**.
+
+Whether you're a domain expert, data scientist, software developer, or anyone in between, Panel provides a flexible and powerful platform for building interactive data apps and dashboards. With its broad range of features and support for multiple visualization libraries and development environments, **Panel is a unique and powerful tool that can help you bring your data to life**.
```

Comment:
Overall I like this description, one thing I don't think it conveys is that it allows building more complex applications than many of our competitors.

Reply:
How would you formulate that?

Reply:
I've updated the home page. Need a new review on this section.